src/ol_infrastructure/applications/airbyte/__main__.py

```diff
@@ -147,6 +157,118 @@
     vpc_id=vpc_id,
 )
 
+airbyte_db_security_group = ec2.SecurityGroup(
+    f"airbyte-db-security-group-{env_name}",
+    name=f"airbyte-db-{target_vpc_name}-{env_name}",
+    description="Access from airbyte to its own postgres database.",
+    ingress=[
+        ec2.SecurityGroupIngressArgs(
+            security_groups=[
+                airbyte_server_security_group.id,
+                consul_security_groups["consul_server"],
+                vault_stack.require_output("vault_server")["security_group"],
+            ],
+            cidr_blocks=[target_vpc["cidr"]],
+            protocol="tcp",
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
+            description="Access to Postgres from Airbyte nodes.",
+        ),
+    ],
+    tags=aws_config.tags,
+    vpc_id=vpc_id,
+)
+
+#########################
+#     Database Setup    #
+#########################
+rds_defaults = defaults(stack_info)["rds"]
+rds_defaults["instance_size"] = (
+    airbyte_config.get("db_instance_size") or rds_defaults["instance_size"]
+)
+rds_password = read_yaml_secrets(
+    Path(f"data/airbyte/rds.{stack_info.env_suffix}.yaml")
+)["password"]
+
+airbyte_db_config = OLPostgresDBConfig(
+    instance_name=f"airbyte-db-{stack_info.env_suffix}",
+    password=rds_password,
+    storage=airbyte_config.get("db_capacity"),
```

Comment:
Is this an optional or required attribute? If you want it to fail if it's not set you can use `airbyte_config.require`